### ISSUE TITLE
Change color/text on some lines/arrows

### ISSUE THREAD

**Issue by Badhumvee** [adrai/flowchart.js#64]
Hi Adrai,
Thanks for creating this awesome tool!  

I have some flowcharts where I need to add some text along some of the arrows (like labels along the arrow) and some where I need to display the arrow using a specific/different color.  I see in your example that you are able to add text to the arrows going out of a condition box, but did not see any way to add a text for an arrow going out of an operation box, which is what I need to do.  Can you please advise how I might be able to add text label to an arrow after operation box and to maybe change its color as well.

Once again, thanks a lot for creating this.  
Cheers!

**Comment by adrai** [comment]
You're right at the moment only arrows going out of a condition box can have text (normally for yes and no paths)...
sorry :-(

**Comment by Badhumvee** [comment]
Can we atleast change the color of a specific arrow?   I am no good at javascript, but hoping you can add this feature to this awesome library.

**Comment by adrai** [comment]
Right now I have no time to do this...
If you know somebody who wants to contribute... everybody is welcome

**Comment by Badhumvee** [comment]
Thanks. Cheers!

**Comment by Badhumvee** [comment]
Hi Adriano,
Two quick questions please:
1) Is it possible to put the arrow head on the connectors in the middle so multiple arrows coming in to same box do not show up only with one arrowhead?
2) The text that I put inside the operation box seems to be sensitive to certain characters.  I had some text with ';' and '()' and it truncated all the following text characters  after those characters.  I understand you are using those characters to separate the meta properties, etc.  Is there a way to use like a escape signature (i.e. /n)to print those special characters in the text box anyway?   Also, can you please send the list of such characters which should be avoided inside the text box?  
Thanks a lot.  Cheers!

**Comment by adrai** [comment]
Hi!

1) not possible right now :-(

2) v1.4.2 ;-)

**Comment by Badhumvee** [comment]
Where do I find the v1.4.2 ?  I dont see it on the website or on github.  Or is it not out yet? Thanks.

**Comment by Badhumvee** [comment]
I found it.  Thanks.

**Comment by adrai** [comment]
At the moment it's just the latest, but you can also follow the git tags



Il giorno 4-ott-2015, alle ore 19:39, Badhumvee <notifications@github.com<mailto:notifications@github.com>> ha scritto:

Where do I find the v1.4.2 ?  I dont see it on the website or on github.  Or is it not out yet? Thanks.

**Comment by rajcitystar** [comment]
Can we have sample project for this?

**Comment by rajcitystar** [comment]
Can we have sample project for this?

**Comment by rajcitystar** [comment]
Can we have sample project for this?
